src/lib/libwebgl.js

```diff
@@ -1820,15 +1820,15 @@ for (/**@suppress{duplicate}*/var i = 0; i <= {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
   },
 
   glGenBuffers: (n, buffers) => {
-    GL.genObject(n, buffers, 'createBuffer', GL.buffers
+    GL.genObjects(n, buffers, GLctx.createBuffer, GL.buffers
```

Comment:
I believe this will pass `createBuffer` as an unbound function, so when it's called `this` will be `undefined` and you'll get a crash, like so:

![image](https://github.com/user-attachments/assets/9c208a0a-4b70-4d0a-b5b1-c76a669d23a5)

Reply:
You could do `() => GLctx.createBuffer()`
or `GLctx.createBuffer.bind(GLctx)`

Reply:
Fair enough.  I guess maybe I'll just abandon this then.